Core/GDCore/Extensions/Builtin/BaseObjectExtension.cpp

```diff
@@ -222,7 +222,7 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsBaseObjectExtension(
                 _("Change the angle of rotation of an object (in degrees)."),
                 _("the angle"),
                 _("Angle"),
-                "res/actions/direction24.png",
+                "res/actions/direction24_black.png",
```

Comment:
should this become rotate24_black too?

Reply:
I kept this icon for actions/conditions that only mention the angle of something, but it could be changed